docs/Installation/InstallationOnAppleSilicon.md

```diff
@@ -158,21 +161,26 @@ cmake \
 -D BUILD_SHARED_LIBS=ON \
 -D MEMORY_ALLOCATOR=SYSTEM \
 -D CHARM_ROOT=${SPECTRE_DEPS_ROOT}/charm/multicore-darwin-arm8 \
+-D SPECTRE_FETCH_MISSING_DEPS=ON \
```

Comment:
I guess it's up to @geoffrey4444 if he wants the docs default to install extra dependencies in people's build directories

Reply:
I think fetching missing dependencies is fine